### DISCUSSION TITLE
Migrate solr package

### DISCUSSION THREAD

**Discussion by pauby** [chocolatey-community/chocolatey-packages#2114] [Migrations]
As per the [migration guidelines](https://github.com/chocolatey-community/chocolatey-packages/wiki/Package-migration-process) I'd like to request migrating the [`solr`](https://community.chocolatey.org/packages/solr) package to here.

The questions to be answered in the doc are:

- _The name of the package that is requested to be migrated:_ `solr`.
- _Why it should be migrated to this repository:_ My goal when picking up this package was to publish the missing versions and then migrate it to this repository where it can be maintained by a team who can keep it up-to-date.
- _A link to both the package and the repository the source code is located in (if it exists):_ https://github.com/pauby/ChocoPackages/tree/master/automatic/solr. As I never intended to keep the package there is no `update.ps1` file in the repository. There is one [here](https://github.com/chocolatey-community/chocolatey-packages/compare/master...pauby:chocolatey-coreteampackages:add-solr-package#diff-2c376cb2d6d5cdceefa908ab306c29c8e267be07810fb66d2e80b28cd84e6b22) ready to submit as part of the [migration PR](https://github.com/chocolatey-community/chocolatey-packages/compare/master...pauby:chocolatey-coreteampackages:add-solr-package).
- _Who will be responsible for fixing issues, reviewing changes, and making sure it is kept up to date (ie, fix the updater when it fails):_ I hadn't planned to add myself to this as it is a fairly straightforward package to update. I can, however, add myself to the `CODEOWNERS` file is the team feels that would be best.

**Comment by pauby** [comment]
After speaking with the team it was agreed to accept this package.

**Reply by AdmiringWorm** [reply]
To confirm, a discussion was done internally with the team members of this repository, and it was decided to move the solr package to this repository.
